eventMacro.pl

```diff
@@ -92,10 +92,13 @@ sub commandHandler {
 		message 
 			"eventMacro MACRO: run macro MACRO\n".
 			"eventMacro list: list available macros\n".
-			"eventMacro status: shows current status\n".
-			"eventMacro stop: stop current macro\n".
-			"eventMacro pause: interrupt current macro\n".
-			"eventMacro resume: resume interrupted macro\n".
+			"eventMacro macro_status: shows current macro status\n".
+			"eventMacro macro_stop: stop current macro\n".
+			"eventMacro macro_pause: pauses current macro execution\n".
+			"eventMacro macro_resume: resumes interrupted macro\n".
+			"eventMacro automacro_checking_status: shows current automacro checking status\n".
+			"eventMacro automacro_checking_pause: pauses automacro checking\n".
```

Comment:
Maybe make this `eventMacro disable automacros`? That sounds a little better (to me).